roles/confluent.zookeeper/tasks/main.yml

```diff
@@ -49,13 +49,17 @@
 - name: Create Zookeeper Group
   group:
     name: "{{zookeeper_group}}"
+  tags:
+    - privileged
 
 - name: Create Zookeeper User
   user:
     name: "{{zookeeper_user}}"
     comment: "Zookeeper User"
     system: true
     group: "{{zookeeper_group}}"
+  tags:
```

Comment:
One thing i was interested in seeing in this PR is to add
```
  become: yes
  become_user: root
```
on any of the privileged tasks, and then in your inventory file we no longer need 
```
ansible_become: true
```